choose-adventure-collection/magicschool/main_app/forms.py

```diff
@@ -0,0 +1,20 @@
+from django import forms
+from .models import Adventure
+
+class AdventureCreateForm(forms.ModelForm):
+    class Meta:
+        model = Adventure
+        fields = ['title', 'description']
+
+    def clean_title(self):
```

Comment:
This is for the filter that has been implement. Allows for the user to safely input the title of their adventure(s). It will automatically capitalize the title(s) on the client side.